datasets/1.1/huggingface-wildchat/metadata.json

```diff
@@ -0,0 +1,1538 @@
+{
```

Comment:
Those tests are likely not run ([source](https://github.com/mlcommons/croissant/blob/f031d966a44b7f9a7037070f8a5a673dee25cc17/python/mlcroissant/mlcroissant/_src/tests/versions.py#L11)). Can you add `1.1` to the list of tested versions?

Reply:
We can't yet, because we do not have 1.1-compliant datasets for all the current datasets we have -- this means that mlcroissant/_src/datasets_test.py::test_static_analysis , mlcroissant/_src/datasets_test.py::test_hermetic_loading , mlcroissant/_src/datasets_test.py::test_raises_when_the_record_set_does_not_exist , ... etc would fail.

We are including 1.1 datasets, however, in mlcroissant/mlcroissant/_src/core/json_ld_test.py

Reply:
Can you add a dedicated test for 1.1? Similar to [this test for 1.0](https://github.com/mlcommons/croissant/blob/f031d966a44b7f9a7037070f8a5a673dee25cc17/python/mlcroissant/mlcroissant/_src/datasets_test.py#L220-L234).